runconfig/opts/parse.go

```diff
@@ -219,7 +219,7 @@ func AddFlags(flags *pflag.FlagSet) *ContainerOptions {
 	flags.BoolVar(&copts.noHealthcheck, "no-healthcheck", false, "Disable any container-specified HEALTHCHECK")
 
 	// Resource management
-	flags.Uint16Var(&copts.blkioWeight, "blkio-weight", 0, "Block IO (relative weight), between 10 and 1000")
+	flags.Uint16Var(&copts.blkioWeight, "blkio-weight", 0, "Block IO (relative weight), between 10 and 1000 (Default is 0 means none)")
```

Comment:
We could use a similar approach as [memory-swap](https://github.com/docker/docker/pull/27947/files#diff-555969c76e0ae264bc1e3c8e880cb50cR242), something like:

```
Block IO (relative weight), between 10 and 1000, or 0 to disable (default 0)
```